README.md

```diff
@@ -83,122 +73,39 @@ yarn build
 
 This will generate production code in the `dist/` directory ready to be used in your nodejs or browser applications.
 
-### Usage
+## Usage
 
-Hello World
+### Hello World example
 
-```javascript
+```ts
 import { TaskExecutor } from "@golem-sdk/golem-js";
 
-(async function main() {
-  const executor = await TaskExecutor.create("9a3b5d67b0b27746283cb5f287c13eab1beaa12d92a9f536b747c7ae");
+(async () => {
+  const executor = await TaskExecutor.create("golem/alpine:latest");
   await executor.run(async (ctx) => console.log(await ctx.run("echo 'Hello World'")).stdout);
   await executor.shutdown();
-})();
+})().catch((err) => console.error(err, "Error in main"));
 ```
```

Comment:
I'm just wondering if this snippet shouldn't look like `hello.mjs` from our `examples` directory?
```js
import { TaskExecutor } from "@golem-sdk/golem-js";

(async function main() {
  const executor = await TaskExecutor.create("golem/alpine:latest");
  try {
    await executor.run(async (ctx) => console.log((await ctx.run("echo 'Hello World'")).stdout));
  } catch (error) {
    console.error("Computation failed:", error);
  } finally {
    await executor.shutdown();
  }
})();
```
or maybe it's better to use top level await instead of IIFE .. (?)